src/App.js

```diff
@@ -11,36 +12,187 @@ class App extends React.Component {
+  //   let tempWinner = -1;
+  //   if (this.state.player1Score > this.state.player2Score) {
+  //     tempWinner = 0;
+  //   } else if (this.state.player2Score > this.state.player1Score) {
+  //     tempWinner = 1;
+  //   } else if (this.state.player1Score === this.state.player2Score) {
+  //     tempWinner = 99;
+  //   }
+
+  //   this.setState({
+  //     roundWinner: tempWinner,
+  //   });
+  // };
+
+  render() {
+    //showing the card image
+    const imageCard = this.state.currCards.map(({ name, suit }) => (
+      <img
+        src={require(`./PNG/${name.toLowerCase()}_of_${suit.toLowerCase()}.png`)}
+        alt={`${name} of ${suit}`}
+        style={{ height: "200px" }}
+      />
     ));
 
+    const numRoundsLeft = this.state.cardDeck.length / 2;
+
+    //winner in each round
+    const winnerMessage =
+      this.state.indexWinner === -1
+        ? ""
+        : this.state.indexWinner === 99
+        ? `Tie`
+        : `Player ${this.state.indexWinner + 1} won!`;
+
+    //overall winner in 26 rounds
+    const gameWinnerMessage =
+      this.state.player1Score === this.state.player2Score
+        ? "Tie"
+        : this.state.player1Score > this.state.player2Score
+        ? `🏆Player 1 WON!🏆`
+        : `🏆Player 2 WON!🏆`;
+
     return (
-      <div className="App">
-        <header className="App-header">
-          <h3>High Card 🚀</h3>
-          {currCardElems}
-          <br />
-          <button onClick={this.dealCards}>Deal</button>
-        </header>
+      <div className="container">
+        <div className="row mt-2">
+          <div className="col-10"></div>
+          <div className="col-2 text-right">
+            <p>{numRoundsLeft} rounds left</p>
+          </div>
+          <div className="text-center mt-1 mb-5">
+            <h3>High Card 🚀</h3>
+          </div>
+
+          <div className="col text-center">
+            {imageCard[0]}
+            <p>Player 1</p>
+            <h2>🏆{this.state.player1Score}</h2>
+            <p>Overall 🏆{this.state.overallPlayer1Score} </p>
+          </div>
+
+          <div className="col text-center">
+            {imageCard[1]}
+            <p>Player 2</p>
+            <h2>🏆{this.state.player2Score}</h2>
+            <p>Overall 🏆{this.state.overallPlayer2Score}</p>
+          </div>
+        </div>
+        {console.log("Player 1 Score:", this.state.player1Score)};
+        {console.log("Player 2 Score:", this.state.player2Score)};
+        <div className="row">
+          <div className="col-12 text-center">
+            {numRoundsLeft > 0 ? (
+              <button className="btn btn-dark" onClick={this.dealCards}>
+                Deal
+              </button>
+            ) : (
+              <button className="btn btn-light" onClick={this.reset}>
+                Reset
+              </button>
+            )}
+          </div>
+
+          <h2 className="text-center mt-2">
+            {numRoundsLeft > 0 && winnerMessage}
```

Comment:
If winnermessage is the empty string as above, you render the h2 unnecessarily. I would do it like so:
```suggestion
            {numRoundsLeft > 0 && winnerMessage && 
            (<h2 className="text-center mt-2">{winnerMessage}</h2>)
            }
```